Plugins/Flow.Launcher.Plugin.Program/Programs/Win32.cs

```diff
@@ -254,10 +235,7 @@ private static Win32 ExeProgram(string path)
             {
                 var program = Win32Program(path);
                 var info = FileVersionInfo.GetVersionInfo(path);
-                if (!string.IsNullOrEmpty(info.FileDescription))
-                {
-                    program.Description = info.FileDescription;
-                }
+                program.Description = info.FileDescription;
```

Comment:
hey @taooceros how come we removed the check here

Reply:
Because this is the only place Description has been initialized, so no matter whether it is empty or null, it won't affect result, i think. Also, the creation of title has checked about the null or empty for description.

Reply:
yes you are right 👍  cool thanks

Reply:
Sorry I have checked again, and it seems that my statement is not right. I will add that check back to void null reference.